cmsFormPage.css

```diff
@@ -0,0 +1,28 @@
+* {
```

Comment:
is this the same css as in `cms.css`?

Reply:
Yes, I just renamed cmsFormPage.css as cms.css.

Reply:
agreed - please remove all duplicated code and just turn the 2 css files into one if you can